flink-ml-lib/src/test/java/org/apache/flink/ml/feature/RandomSplitterTest.java

```diff
@@ -95,7 +96,7 @@ public void testOutputSchema() {
     @Test
     public void testWeights() throws Exception {
         Table data = getTable(1000);
-        RandomSplitter splitter = new RandomSplitter().setWeights(2.0, 1.0, 2.0);
+        RandomSplitter splitter = new RandomSplitter().setWeights(2.0, 1.0, 2.0).setSeed(0);
```

Comment:
It seems that the default seed is not good enough to generate split results that meet statistical expectations. Can we modify the random splitting behavior to improve the default splitting behavior?

Reply:
The default seed can good enough to generate split results that meet statistical expectations. I have remove the setSeed(0)